internal/db/models/programs.go

```diff
@@ -0,0 +1,41 @@
+package models
```

Comment:
I am assuming that this is the file that you add structs to in order to create more interaction types, am I correct in assuming that?

Reply:
Yes, more interaction types will be added here, as well as in `panda_studio.yaml` and `.marshal.go`